src/processTranslation.tsx

```diff
@@ -60,14 +60,25 @@ export const processTranslation = ({
     throw new Error(`Missing translation: "${key}" with suffix: "${suffix}"`);
   }
 
+  const {
+    context,
+    ns,
+    ...replaceableArgs
+  } = args;
+
   // Replace placeholders like `{{count}}` with values from `args`
-  const processed = Object.entries(args).reduce(
-    (v, [ name, value ]) => v.replace(
-      new RegExp(`{{\\s*${name}\\s*}}`, 'g'),
-      String(formatArgument(locale, value)),
-    ),
+  return Object.entries(replaceableArgs).reduce(
+    (v, [ name, value ]) => {
+      const regexp = new RegExp(`{{\\s*${name}\\s*}}`, 'g');
+
+      if (!regexp.test(v)) {
+        console.error(`Argument: "${name}" with value: "${value}" is not valid`);
```

Comment:
what do you think about adding a configuration option to the library, and let the user decide what to do in this case?
By doing so you'll empower them to pass `console.error`,`report-somewhere`, or nothing.

Reply:
Maybe create a callback? I mean like an option, so in the app we will be able to pass any function, for example, even Sentry capture.

Reply:
Thank you, nice suggestion, with this we'll send errors to sentry instead of throwing the error or just writing the error in the client console.

Thinking about introducing some breaking changes and executing onError for missing the whole `translation` not just the wrong argument...

Reply:
Discussed that it's better keep throwing the error on the higher level and make soft error handling for arguments